### PR TITLE
chore: update to patch security vulnerabilities caused from dependencies

### PR DESCRIPTION
**Pull request by awentzel**
# Description
This updates a handful of dependencies and one failing unit test.

closes #2871 

## Motivation & context
We have 22 open security alerts on GitHub that require a fix to be compliant.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components/storybook-static/iframe.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    background: #000;
+    color: #eee;
+    font-family: "Operator Mono", "Fira Code Retina", "Fira Code", "FiraCode-Retina", "Andale Mono", "Lucida Console", Consolas, Monaco, monospace;
+  }
+
+  .sb-errordisplay pre {
+    white-space: pre-wrap;
+  }</style><script>/* globals window */
+  /* eslint-disable no-underscore-dangle */
+  try {
```
Comment (nicholasrice) [inline packages/fast-components/storybook-static/iframe.html:62]: You should delete the storybook-static dir. It should be git-ignored

Comment (awentzel) [reply packages/fast-components/storybook-static/iframe.html:62]: If it's not `.gitignored` I'll submit another PR and remove these files from the PR.

File: packages/fast-tooling-react/src/css-property-editor/property-editor.spec.tsx [hunk 2]
```diff
@@ -42,21 +42,21 @@ describe("CSSPropertyEditor", () => {
...
+    //         .at(0)
+    //         .simulate("click");
+
+    //     expect(
+    //         rendered
+    //             .find("input")
+    //             .at(0)
+    //             .getDOMNode()
+    //     ).toEqual(document.activeElement);
+    // });
```
Comment (janechu) [inline packages/fast-tooling-react/src/css-property-editor/property-editor.spec.tsx:59]: Can you just put "x" in front of the test and leave a comment on why the test is disabled? Should look something like this: `xtest("should focus...`

Comment (awentzel) [reply packages/fast-tooling-react/src/css-property-editor/property-editor.spec.tsx]: Thanks for the idea.  Resolved.

File: .gitignore [hunk 3]
```diff
@@ -67,6 +67,7 @@ dist/
 
 # Ignore build www directories
 www/
+storybook-static
```
Comment (nicholasrice) [inline .gitignore:70]: thanks!

**Pull request merged**